fusor/fusor.py

```diff
@@ -536,3 +568,44 @@ def translate_identifier(
         if ga4gh_identifiers:
             return ga4gh_identifiers[0]
         return None
+
+    def _get_sequence_validation_warnings(
```

Comment:
I think you should import the `SeqRepoAccess` class and use `get_sequence` from variaiton-normalization rather than copying the code over into this repo.

Reply:
I think we could just go with a generic error message if `get_sequence` returns nothing. Like `seqrepo unable to get positions on transcript`

Reply:
Is it worth it to add variation-normalizer as a dependency? Generic error messages mean you can't provide meaningful feedback clientside (eg transcript not recognized vs coordinates are outside of the transcript's range)

Reply:
I think we should use it if the code already exists. We can make an issue in variaiton-normalization to return the errors rather than returning None (this will also start to address [this](https://github.com/cancervariants/variation-normalization/issues/49) issue).

I'm thinking about this more and I'm wondering if we should just move the seqrepo access class to uta-tools... I think I'm actually going to do that. We can leave this is as now and once seqrepo access class is moved to uta-tools we can switch to using its method.

Reply:
Gotcha, adding it to UTA tools works for me if we can make warning messages precise